docs/Synthesis UI.md

```diff
@@ -0,0 +1,92 @@
+# UI interactions in various modes
+
+Document describes modes that browser.html could be in & how various UI components will behave in that mode based of the **Synthesis UI / Jun 15, 2015** specification.
+
+## Actions
+
+- `ShowWebView`
+- `ShowWebViewByID`
+- `CreateWebView`
+- `EditWebView`
+- `ChooseNextWebView`
+- `ChoosePreviousWebView`
+- `Escape`
+- `CloseWebView`
+
+## Modes
```

Comment:
We will also need a way to hook in animation transitions for each `mode -> mode.` Transitions between states aren't symmetric. For example

- `show-webview -> create-web-view using zoom out`
- `create-web-view -> show web-view using no animation`
- `show-web-view -> choose-web-view using no animation`
- `choose-web-view -> show-web-view using no animation`
- ...